app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraFragment.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+
+using Android.Support.V4.App;
+using Android.Hardware;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+
+namespace Moodis.Feature.CameraFeature
+{
+    public class CameraFragment : Fragment
```

Comment:
Just a question, why is there a camera activity and a fragment as well? Is it possible to just put everything to Camera Activity?

Reply:
Yeah it's possible as a fragment is already a part of an activity. However having it as a fragment allows us to use easily use camera in any activity as it only needs a fragment view. Also it improves code readability as all camera inner workings is declared in the cameraFragment class.